components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/handlers/AbstractResponseTypeHandler.java

```diff
@@ -96,7 +96,7 @@ && hasValidationByApplicationScopeValidatorsFailed(oauthAuthzMsgCtx)) {
         OAuth2AuthorizeReqDTO authorizationReqDTO = oauthAuthzMsgCtx.getAuthorizationReqDTO();
         OAuthCallback scopeValidationCallback = new OAuthCallback(authorizationReqDTO.getUser(),
                 authorizationReqDTO.getConsumerKey(), OAuthCallback.OAuthCallbackType.SCOPE_VALIDATION_AUTHZ);
-        scopeValidationCallback.setRequestedScope(authorizationReqDTO.getScopes());
+        scopeValidationCallback.setRequestedScope(oauthAuthzMsgCtx.getRequestedScopes());
```

Comment:
Will the behaviour change with this change in getting the scopes passed?

Reply:
perviously all changes happened to scopes in `authorizationReqDTO` in the validation process. now its requestedScopes in `oauthAuthzMsgCtx`.  seems no behaviour change  with this change